### ISSUE TITLE
Release 1.8.144 forces you to use Exchange for SMTP

### ISSUE THREAD

**Issue by stefannitsche** [freescout-help-desk/freescout#4084]
The fix for #4072 seems to have been "a bit too good".  Unfortunately it makes it impossible to use Exchange Online for receiving emails and another service for sending, like Sendgrid, Postmark etc.

I would rather not send emails through Exchange Online and continue using my SMTP-service so some kind of option to actually set if it should use Exchange or not when choosing the SMTP option seems to be needed.

Basically what I'm humbly requesting is a way to somehow toggle between the previous behavior (1.8.143 and earlier)  and the behavior in 1.8.144

**Comment by freescout-help** [comment]
You still can do it: first enable Fetching, after that configure Sending email via your SMTP server.

**Comment by stefannitsche** [comment]
That isn't my experience sadly.

This how my settings look after upgrading: 

![image](https://github.com/freescout-help-desk/freescout/assets/13170419/e4da7f2f-8ce2-4277-8cae-165343ac6dd7)

Except the green Microsoft Exchange everything here looks alright. But when I try to send a test email the issue gets exposed:

![image](https://github.com/freescout-help-desk/freescout/assets/13170419/21dc8471-03ed-4776-b2df-11bcc601d14e)

It seems that is now ignoring my specified username and instead tries to use the "Fetch From" address.  The 0 possible authenticators also seems a bit off but I really can't comment on that.

**Comment by freescout-help** [comment]
You need to enter Username and Password from your SMTP server. When configuring Fetching the system automatically changes SMTP password. So not you need to change it back.

**Comment by stefannitsche** [comment]
No change in behavior if I add the password again. It still tries to use `Fetch From` as username.

The username we receive from our SMTP-provider does not include the `@` symbol. It is a randomized string containing number, letters and dashes, which wasn't an issue in 1.8.143 and has never been an issue anywhere else. It does seem like it only tries to use the provided SMTP-username if includes `@`. But this doesn't help me since that's not how our usernames are formatted. Which then brings me back full cirlce of asking for the option to somehow bringing back the old behavior of 1.8.143.

Our current situation with 1.8.144 is that if I disconnect from Exchange the SMTP works but we can't receive any conversations, if I connect to Exchange the SMTP stops working and we can't reply to any conversations. This less of an ideal situation.

**Comment by freescout-help** [comment]
Try with the attached `/app/Misc/Mail.php` file:
[Mail.zip](https://github.com/user-attachments/files/15903205/Mail.zip)

**Comment by stefannitsche** [comment]
That does indeed work. Awesome job, thank you very much!
